pipeline/test/test_SimulationConfig.jl

```diff
@@ -0,0 +1,15 @@
+
+# Test the TruthSimulationConfig struct constructor
+@testset "TruthSimulationConfig" begin
```

Comment:
why are we not using testitem?

Reply:
Basically, I was keeping the testing very separate from `EpiAware`. Might be worth just being a bit careful and seeing what I can do here?

Reply:
does it not only show up if you are in the environment where those tests lives? Hmm I wonder if there is a good way to filter. Seems very annoying to not be able to use it?

Reply:
Shall wel make this an issue and leave for now?